src/main/java/ru/itmo/sd/deadliner2bot/state/SelectTodoState.java

```diff
@@ -0,0 +1,61 @@
+package ru.itmo.sd.deadliner2bot.state;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.api.methods.BotApiMethod;
+import ru.itmo.sd.deadliner2bot.messages.MessageService;
+import ru.itmo.sd.deadliner2bot.model.Chat;
+import ru.itmo.sd.deadliner2bot.model.ChatStateEnum;
+import ru.itmo.sd.deadliner2bot.model.Todo;
+import ru.itmo.sd.deadliner2bot.repository.ChatRepository;
+import ru.itmo.sd.deadliner2bot.service.TodoService;
+
+import java.util.List;
+import java.util.Optional;
+
+@Component
+@RequiredArgsConstructor
+public class SelectTodoState implements ChatState {
+
+    private final ChatRepository chatRepository;
+    private final ChatStateEnum chatStateEnum = ChatStateEnum.SELECT_TODO;
+    private final TodoService todoService;
+    private final MessageService messageService;
+
+    @Override
+    public List<BotApiMethod<?>> process(Chat chat, String message) {
+        if (message.startsWith("/cancel")) {
+            chat.setState(ChatStateEnum.BASE_STATE);
+            chatRepository.save(chat);
+            return List.of(messageService.createMessage(chat, "Operation cancelled"));
+        } else if (message.startsWith("/select")) {
+            String number = message.substring(8);
+            System.out.println(number);
```

Comment:
Please, get rid of unnecessary `System.out.println`s. If you need some debug info, use `log.debug()`.